lib/seed/drug_lookup_tables_seeder.rb

```diff
@@ -0,0 +1,32 @@
+module Seed
+  class DrugLookupTablesSeeder
```

Comment:
Try `include DrugLookup`

Reply:
Fixed in 3ab2b71aa9719ca105c013ab0f3991512075445f.